docker/Dockerfile

```diff
@@ -108,6 +108,8 @@ RUN apt-get update && \
       localedef -i en_GB -c -f UTF-8 -A /usr/share/locale/locale.alias en_GB.UTF-8
 
 
+COPY --from=samtools_build /usr/local/bin/plot-ampliconstats /usr/local/bin/plot-ampliconstats
```

Comment:
Don't these require gnuplot to work? I can't see that being installed in the Dockerfile

Reply:
Indeed it is. Thanks!

For my future reference, `plot-ampliconstats  --help` does not complain about the missing _Gnuplot_.

_Gnuplot_ should be included in next commit. Would like to manually squash commits, so it is atomic change. @kjsanger if you are happy with the next commit, please give me a chance to rebase before merging.